tap_github_search/search_count_streams.py

```diff
@@ -257,14 +253,11 @@ def _get_repo_counts_via_batching(self, repos: list[str], org: str, rest_query:
                 self.logger.warning(f"Batch failed, falling back to individual queries: {str(e)}")
```

Comment:
Catch exceptions of specific types in the line above

Reply:
We are catching [`requests.exceptions.RequestException` ](https://requests.readthedocs.io/en/latest/_modules/requests/exceptions/) now.